oeps/oep-0021-proc-deprecation.rst

```diff
@@ -22,295 +22,323 @@ OEP-0021: Deprecation and Removal
+   **Migration** section.
+#. If there is any additional publicly shareable information or data from your
+   earlier analysis, include that on the ticket.
 
-   Subject: Deprecation Announcement: [*Technology Name*]
+Now you are ready to communicate your proposal.
 
-   Body: 
-         This message is to inform the Open edX community that we intend to
-         deprecate and remove [*Technology Name*] from the Open edX platform.
-         This technology currently does the following: [*Description*] and is
-         found in the following places [*Repo name(s) or other locations*].
-         Deprecation of this techology will affect the following users: 
-         [*Affected users*]
+.. _Deprecation and Removal JIRA board: https://openedx.atlassian.net/secure/RapidBoard.jspa?rapidView=452
+.. _DEPR-6: https://openedx.atlassian.net/browse/DEPR-6
 
-         We expect that this process will happen over the following timeline:
-         [*Timeline*] and if there are any questions or concerns, please reach
-         out to [*Point of Contact*].
+Communicated
+------------
 
-Status tracking
----------------
+Announce
+~~~~~~~~
+Announce your proposal to deprecate and remove to the following communication
+channels.
 
-.. image:: oep-0021/state-flow.png
-   :alt: A diagram that shows the state flow transitions. The process starts in the Proposed state and goes
-    through the Communicated, Accepted, Deprecated, Removing, and Removed states. If the proposal isn't Accepted, 
-    the state transitions from Proposed to Abandoned.
+To edx-code
+^^^^^^^^^^^
 
-JIRA ticket states (with suggested information to include in ticket
-description):
+Send an email to edx-code_ using the following template, bcc-ing your own
+engineering organization (`edx-code example`_):
 
--  Proposed
+    To: edx-code@googlegroups.com
 
-   -  Include target dates for each subsequent step.
+    Subject: Deprecation/Removal: <*Technology Name*> <*DEPR-Number*>
 
--  Communicated
+    Body: 
+        Hi there,
 
-   -  Include link to communication on edx-code.
+        We plan to deprecate and remove <*Short description of the technology*>.
+        We are targeting removal by <*Target Removal Date*>.
 
-   -  Accept feedback/input from community until target date for “Deprecated”.
+        Please read https://openedx.atlassian.net/browse/<*DEPR-Number*> for
+        more information and to post any questions/comments. The proposed
+        deadline for comments before acceptance is <*Target Accepted Date*>.
 
-   -  Adjust target dates, as needed.
+        Thanks,
+        <*Your name*>
 
--  Accepted
+.. _edx-code: https://groups.google.com/forum/#!forum/edx-code
+.. _edx-code example: https://groups.google.com/d/msg/edx-code/779kAKas2Yw/jU7vGHu8CgAJ
 
-   -  Community feedback has been addressed and there is alignment on timeline.
+To openedx.slack.com
+^^^^^^^^^^^^^^^^^^^^
 
--  Deprecated
+Post the following in the #open-edx-proposals and #dev `openedx slack`_ channels:
```

Comment:
Just curious: why #dev and not, say, #general or #ops?

Reply:
hmm.. someone recommended #dev to me (as it corresponded with an internal edX dev channel) but if these others are more appropriate, then we can change this.

@nedbat @mduboseedx  What do you think are the appropriate openedx slack channels for announcing deprecation and removals?

Reply:
@pdpinch I confirmed with the open edX team internally - and they agree with you that #general is more appropriate for this.  Thanks for pointing it out!  I'll update the doc.